docs/Windows.md

```diff
@@ -3,25 +3,55 @@
 These instructions cover building 32blit on Windows (without WSL).
 
 - [Prerequisites](#prerequisites)
-- [Using Visual Studio](#using-visual-studio)
+- [Setting Up](#setting-up)
 - [Building & Running on 32Blit](#building--running-on-32blit)
 - [Building & Running Locally](#building--running-locally)
   - [Build Everything](#build-everything)
+- [Troubleshooting](#troubleshooting)
+  - [Multiple definition of `cleanup_glue`, `_reclaim_reent` etc](#multiple-definition-of-cleanup_glue-_reclaim_reent-etc)
+
+To build using Visual Studio [see here](Windows-VisualStudio.md). The rest of these instructions cover command-line builds.
 
 ## Prerequisites
 
-You'll need to install:
+- Windows Terminal - grab it from the Windows Store
+- [Git for Windows](https://git-scm.com/download/win)
+- [Python](https://www.python.org/downloads/) - eg: "python-3.9.6-amd64.exe"
+- The [GNU Arm Embedded Toolchain](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads) (For 32blit device builds, use the 9-2020-q2-update version). Make sure to select the "Add path to environment variable" at the end of the setup.
+ - The [Visual Studio build tools](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) (For local builds, select "C++ build tools"). This includes CMake and NMake.
+
+If you have Visual Studio installed you can Modify the installation and select "C++/CLI support for vXXX build tools (Latest)"
+
+## Setting Up
+
+*Note: Developer Command Prompt is a little clunky but you can add it to Windows Terminal with the following config:*
+
+```json
+{       
+    "commandline": "cmd.exe /k \"C://Program Files (x86)//Microsoft Visual Studio//2019//Community//Common7//Tools//VsDevCmd.bat\"",
+    "cursorColor": "#EEEEEE",
+    "cursorShape": "bar",
+    "fontFace": "Consolas",
+    "fontSize": 10,
+    "guid": "{5ee0706e-b015-46b2-98a3-2122a8e627d3}",
+    "historySize": 9001,
+    "icon": "C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\Common7\\IDE\\Assets\\VisualStudio.70x70.contrast-standard_scale-80.png",
+    "name": "Developer Command Prompt for VS2019",
+    "padding": "0, 0, 0, 0",
+    "snapOnInput": true,
+    "startingDirectory": "%USERPROFILE%"
+}
+```
```

Comment:
Used
```json
            {       
                "commandline": "cmd.exe /k \"C://Program Files (x86)//Microsoft Visual Studio//2022//BuildTools//Common7//Tools//VsDevCmd.bat\"",
                "guid": "{5ee0706e-b015-46b2-98a3-2122a8e627d3}",
                "name": "Developer Command Prompt for VS2022",
                "startingDirectory": "%USERPROFILE%"
            }
```

for VS 2022 build tools. (Couldn't find anything to use as an icon and the rest seems optional)